lib/page/questions/widget/questions_item_widget.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:survey/model/question_model.dart';
+import 'package:survey/resource/dimens.dart';
+import 'package:survey/widget/dimmed_background_widget.dart';
+import 'package:survey/widget/next_button_widget.dart';
+import 'package:survey/widget/rounded_button_widget.dart';
+
+class QuestionsItemWidget extends StatelessWidget {
```

Comment:
This widget contains only 1 question, the name should not in the plural. How about `QuestionPageWidget`?